modules/csp-overview.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plugin/content-security-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="content-security-policy-overview_{context}"]
+= Content Security Policy (CSP) overview
+
+A Content Security Policy (CSP) is delivered to the browser in the `Content-Security-Policy-Report-Only` response header. The policy is specified as a series of directives and values. Each directive type serves a different purpose, and each directive can have a list of values representing allowed sources.
+
+[id="content-security-policy-key-features_{context}"]
+== Key features of `ConsolePluginCSP`
+
+[discrete]
+=== Directive Types
+
+The supported directive types include `DefaultSrc`, `ScriptSrc`, `StyleSrc`, `ImgSrc`, and `FontSrc`. These directives allow you to specify valid sources for loading different types of content for your plugin. Each directive type serves a different purpose. For example, `ScriptSrc` defines valid JavaScript sources, while `ImgSrc` controls where images can be loaded from.
```

Comment:
```suggestion
The supported directive types include `DefaultSrc`, `ScriptSrc`, `StyleSrc`, `ImgSrc`, `FontSrc` and `ConnectSrc`. These directives allow you to specify valid sources for loading different types of content for your plugin. Each directive type serves a different purpose. For example, `ScriptSrc` defines valid JavaScript sources, while `ImgSrc` controls where images can be loaded from.
```

Reply:
We have new `ConnectSrc` added in [4.19 support PR](https://github.com/openshift/api/pull/2163) and [4.18 support PR](https://github.com/openshift/api/pull/2164)
4.18 PR is still open, so if this doc is targeted for latest console then it would be safe to add `ConnectSrc` , otherwise we need consider when the support for 4.18 can be merged

Reply:
I will keep an eye on the PR and add it into the docs once the PR is merged. Thanks!